cfme/tests/cloud/test_providers.py

```diff
@@ -96,6 +96,38 @@ def test_providers_discovery_amazon(appliance):
     wait_for_a_provider()
 
 
+@pytest.mark.uncollectif(lambda provider: (store.current_appliance.version >= '5.9' or
```

Comment:
**QUESTION**
Why do we need a 5.8 only test?

Reply:
@quarckster :
The test is for cloud provider discovery. The discovery feature is support for 5.8 but not in 5.9. They just removed discovery option for cloud provider l.e. both Azure and AWS.

Reply:
**Required**

Use `provider.appliance.version` instead of store.current_appliance